langchain4j-bom/pom.xml

```diff
@@ -215,6 +215,12 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>dev.langchain4j</groupId>
+                <artifactId>langchain4j-mcp</artifactId>
```

Comment:
Maybe it should be `langchain4j-mcp-client`?

Reply:
That depends, do we plan to have the server module in langchain4j too? I think that would probably be an independent project. Also, the module doesn't just provide the 'client' per se, it also provides a `ToolProvider` for it

Reply:
> That depends, do we plan to have the server module in langchain4j too?

Very unlikely. 

> Also, the module doesn't just provide the 'client' per se, it also provides a ToolProvider based on it (and more stuff in the future)

Okay, makes sense